### ISSUE TITLE
Issue with page translated fields migrations

### ISSUE THREAD

**Issue by marctc** [infoportugal/wagtail-modeltranslation#35]
I added a translatable field to a model that I previously defined. When I run `makemigrations` of my app, It creates these migrations:
```
Migrations for 'wagtailcore':
  0020_auto_20150930_1205.py:
    - Add field search_description_ca to page
    - Add field search_description_en to page
    - Add field search_description_es to page
    - Add field seo_title_ca to page
    - Add field seo_title_en to page
    - Add field seo_title_es to page
    - Add field slug_ca to page
    - Add field slug_en to page
    - Add field slug_es to page
    - Add field title_ca to page
    - Add field title_en to page
    - Add field title_es to page
    - Add field url_path_ca to page
    - Add field url_path_en to page
    - Add field url_path_es to page
```
that seems a default behaviour of wagtail-modeltranslation. This is not quite good because if I have to deploy this project these migrations won't be deployed as are located in my local site-packages.

A common solution for this is specifying in `setings.py` that migrations would be on my app: 
```python
    MIGRATION_MODULES = {'wagtailcore': 'cms.migrations_wagtail'}
```
But this its seem that it isn't working properly:
```
django.db.migrations.graph.NodeNotFoundError: Migration wagtailredirects.0001_initial dependencies reference nonexistent parent node ('wagtailcore', '0002_initial_data')
```
Any idea of how to fix this?

**Comment by SalahAdDin** [comment]
Same here man,

**Comment by fdemmer** [comment]
@marctc when you change the path via `MIGRATION_MODULES`, you have to make sure the new path contains *all* the previous migrations. so in this case copying the old wagtail migrations from `site-packages` into your project under `cms/migrations_wagtail` should fix the problem.

**Comment by marctc** [comment]
I've already realized that your solution fixes the problem, but it's a overkill do it in this way: I have to move all Wagtail migrations and his dependencies. What if I want to upgrade to a new Wagtail version? Could be messy process.

**Comment by rmartins90** [comment]
The solution is to get rid of any modifications on Page model directly. Soon i'll commit a solution.

**Comment by marctc** [comment]
:+1:

**Comment by rmartins90** [comment]
Fixed with 59bc9e5
